src/sql/base/browser/ui/radioButton/radioButton.ts

```diff
@@ -33,7 +35,18 @@ export class RadioButton extends Widget {
 		this.label = opts.label;
 		this.enabled = opts.enabled || true;
 		this.checked = opts.checked || false;
-		this.onclick(this.inputElement, () => this._onClicked.fire());
+		this.onclick(this.inputElement, () => {
+			this._onClicked.fire();
+			let buttonGroup = document.getElementsByName(this.name);
+			buttonGroup.forEach((button) => {
+				let event = document.createEvent('HTMLEvents');
+				event.initEvent('change', true, true);
+				button.dispatchEvent(event);
```

Comment:
We should avoid sending the event if the checked state hasn't actually changed

Reply:
Won't the state be change for all the buttons with the same "name" attribute?

Reply:
Only two - the previously checked one and the newly checked on.